source/dynamic_data_attachment.md

```diff
@@ -0,0 +1,97 @@
+# Dynamic Data Attachment
+
+This support article illustrates how data from one project can be linked dynamically and referenced by another project(s). It also aims to demonstrate some practical examples that this feature supports.
+
+Now that this feature is available with KoBoToolbox, users can do everything that a [pulldata function](pull_data_kobotoolbox.md) does without having to maintain a separate CSV file. The data stored in the survey project (that the user intends to link) acts as a source file. Users could keep collecting data for their source files and simultaneously use them in another survey project(s). Thus, this feature should help users create and manage a longitudinal survey project _(longitudinal data is collected through a series of repeated observations of the same subjects over a short or long period)_. The best part of this feature is that it also works with a shared project where the project admin could also restrict specific permissions while sharing.
+
+Users can now retrieve any information (such as _text_, _integer_, _decimal_, _select_one response_, _select_multiple response_, _GPS points_) to a child project (aka round 2 project) that has already been collected and stored in a parent project (aka round 1 project). However, media information (such as _image_, _audio_, _video_, _uploaded files_, etc.) cannot be retrieved at the moment.
+
+At the same time, users should also be able to perform various calculations such as counting responses entered and stored in a parent project directly through the child project. Similarly, users are also able to compute the _sum_, _maximum_, or _minimum_ value of a particular variable collected and stored in a parent project directly through a child project.
+
+## Linking a project dynamically through xlsform:
+
+While trying this approach through the xlsform, users will require two xlsform. The first xlsform (also known as <a download href="./_static/files/dynamic_data_attachment/Round_1.xlsx"> parent form or round 1 form</a>) is generally a normal xlsform that we often design. The second xlsform (also known as <a download href="./_static/files/dynamic_data_attachment/Round_2.xlsx"> child form or round 2 form</a>) is slightly different from the first xlsform, especially in two aspects, i.e., it has an xml-external question type, and it also has a calculation column that holds the path to the survey project being linked.
+
+Users could design their parent project as shown in the image below:
+
+![image](/images/dynamic_data_attachment/Round_1.png)
+
+In the same way, users could design their child project as shown in the image below:
+
+![image](/images/dynamic_data_attachment/Round_2.png)
+
+__Note:__ The name used for the _xml-external_ question type in the child project is crucial for linking the project with the parent project. In the above example, it was named survey. You can give it a different name as well _(it can only consist of Latin characters and numbers)_.
```

Comment:
Rather use `xml-external`

Reply:
_child project_ ... _parent project_

Reply:
> In the above example, it was named survey. You can give it a different name as well _(it can only consist of Latin characters and numbers)_.

Rather

> In the above example, it was named "survey", but it can be any name consisting of Latin characters and numbers.